pkgs/sdk/client/src/DataModel.cs

```diff
@@ -128,6 +154,14 @@ public static FeatureFlag ReadJsonValue(ref Utf8JsonReader reader)
                         case "debugEventsUntilDate":
                             debugEventsUntilDate = JsonSerializer.Deserialize<UnixMillisecondTime?>(ref reader);
                             break;
+                        case "prerequisites":
+                            for (var array = RequireArrayOrNull(ref reader); array.Next(ref reader);)
+                            {
+                                prerequisites ??= new List<string>();
+                                prerequisites.Add(reader.GetString());
+                            }
```

Comment:
Doing the null coalescing operator because otherwise, we'd potentially construct an empty prereq array if the JSON value is null.